systemtest/src/main/java/io/strimzi/systemtest/resources/crd/KafkaResource.java

```diff
@@ -204,6 +204,7 @@ public static Kafka kafkaWithoutWait(Kafka kafka) {
      */
     public static void deleteKafkaWithoutWait(Kafka kafka) {
         kafkaClient().inNamespace(ResourceManager.kubeClient().getNamespace()).delete(kafka);
+        ResourceManager.waitForKafkaDeletion(kafka);
```

Comment:
but now it is no longer without wait you are waiting until Kafka will be deleted

Reply:
but this is on the end of a test -> if you don't delete it on the end of a test, it can affect the next one -> maybe we should add new method for deleting Kafka with wait (I thought this is just name for deleting Kafka that is created without wait, that's it)

Reply:
how we discussed on chat, I'll add another one method that will also wait for deletion, not just delete it

Reply:
In general you are right. Kafka creation without wait is used in tests, where we expect that Kafka will not start properly. Have there wait for deletion should be fine, because we won't to delete it properly in case of any troubles or fail tests if CR will not be deleted.